content/Classes/Classes.facts.md

```diff
@@ -0,0 +1,21 @@
+### Facts for Classes lesson:
+
+A class is a blueprint that defines the characteristics and behaviors of all objects of a specific kind. It has properties and methods as its basic building blocks.
+
+A property is a variable that is defined inside a class. 
+
+A method is a function inside a class that performs a specific action.
+
+To declare a class, use the `class` keyword followed by the class name, and a pair of curly braces `{}` that enclose the definition of the `properties` and `methods` of a class. 
+
+The following is an example of a class declaration:
+
+```php
+class MyClass {
+    // properties and methods here...
+}
+```
+
+A class name may start with a letter or an underscore `_`, followed by any combination of alphanumeric characters and underscores (`A-Z`, `a-z`, `0-9`, `_`). 
+
+It is good practice to capitalize the first letter of a class name to easily identify them from other variables. Some examples of good class names are `MyClass`, `SuperHero`, and `Car`.
```

Comment:
Yes, this is the reason why I included this statement but I did not make the statement too strong since there is no specific rules about this. Thank you.

Reply:
hmm, ok.